apollo-client/apollo_client.go

```diff
@@ -100,7 +100,9 @@ func (apolloClient *ApolloClient) PullConfigs(serviceName, version, app, env str
 
 	lager.Logger.Debugf("The Marshaled response of the body is : ", configurations["configurations"])
 
-	return configurations, nil
+	var configValues map[string]interface{}
```

Comment:
why convert it

Reply:
Configurations comes with map[string] interface which has configurations["configurations"] as an interface and some other metadata. Our configcenter source expects this configurations[configurations"] as a map[string] interface so we have to convert it
````
Sample Response from Apollo Server
		{
			"appId": "SampleApp",
			"cluster": "default",
			"namespaceName": "application",
			"configurations": {
				"timeout": "500"
			},
			"releaseKey": "20180327130726-1dc5027439679153"
		}
```